build.fsx

```diff
@@ -411,13 +416,22 @@ Target.Create "DotNetCoreIntegrationTests" (fun _ ->
     |> NUnit3.NUnit3 id
 )
 
+#if BOOTSTRAP
+let withWorkDir wd (cliOpts: DotNet.Options) = { cliOpts with WorkingDirectory = wd }
+#else
 let withWorkDir wd (cliOpts:Cli.DotNetOptions) = { cliOpts with WorkingDirectory = wd }
+#endif
 
 Target.Create "DotNetCoreUnitTests" (fun _ ->
     // dotnet run -p src/test/Fake.Core.UnitTests/Fake.Core.UnitTests.fsproj
+    #if BOOTSTRAP
+    let processResult =
+        DotNet.Raw (withWorkDir root) "src/test/Fake.Core.UnitTests/bin/Release/netcoreapp2.0/Fake.Core.UnitTests.dll" "--summary"
```

Comment:
Or `DotNet.Exec` (but only a suggestion, it is as good as `Raw`, I just think we used `Exec` already on some places).
On the other side we also used `Run` on other places. This is indeed not a very consistent naming at the moment.

Reply:
I find `Exec` a good name.

Indeed using `Run` seems most natural but their is a `run` command in dotnet so it's can be confusing.